monero-outreach-docs/translations/zh-hans/three_reasons/three_reasons_zh-hans.md

```diff
@@ -0,0 +1 @@
你需要隐私以避免不幸的常见风险，如身份信息被盗、通过广告操纵你、基于个人信息的歧视、骚扰、过滤气泡，以及侵犯隐私带来的许多其他实际伤害。

In addition, what many people don’t realize is that several small pieces of your personal data can be put together to reveal much more about you than you would think is possible. For example, an analysis conducted by MIT researchers found that “just four fairly vague pieces of information — the dates and locations of four purchases — are enough to identify 90 percent of the people in a data set recording three months of credit-card transactions by 1.1 million users.”
此外，许多人没有意识到的是，你的几个小的个人数据可以组合在一起，透露的信息要比你想象的多得多。例如，麻省理工学院的研究人员进行的一项分析发现，“根据110万用户3个月信用卡的交易记录，只要4条相当模糊的信息，4条购物的数据及地点，就足够在一个数据库中识别出90%的人。”

It’s critical to remember that privacy isn't just about protecting a single and seemingly insignificant piece of personal data, which is often what people think about when they say, “I have nothing to hide.” For example, some may say they don't mind if a company knows their email address while others might say they don't care if a company knows where they shop online.
重要的是要记住，隐私不仅仅是保护一个看似无关紧要的个人数据。当人们说“我没有什么好隐瞒的”时，他们往往会这样想。例如，一些人可能会说他们不介意公司知道他们的电子邮件地址，而另一些人可能会说他们不介意公司知道他们在哪里网上购物。

However, these small pieces of personal data are increasingly aggregated by advertising platforms like Google and Facebook to form a more complete picture of who you are, what you do, where you go, and with whom you spend time. And those large data profiles can then lead much more easily to significant privacy harms. If that feels creepy, it’s because it is.
然而，像谷歌和Facebook这样的广告平台越来越多地收集这些小块的个人数据，以便更全面地了解你是谁、你在做什么、你去哪里、你和谁在一起。而这些大型数据档案则更容易导致严重的隐私损害。如果这让人觉得毛骨悚然，那是因为确实如此。


We can't stress enough that your privacy shouldn’t be taken for granted. The ‘I have nothing to hide’ response does just that, implying that government and corporate surveillance should be acceptable as the default.
我们再怎么强调也不为过，你的隐私不应该被毫无理由地暴露出来。“我没有什么好隐瞒的”就是对这种想法的回应，这意味着政府和企业的监控应该作为默认情况被接受。

Privacy should be the default. We are setting a new standard of trust online and believe getting the privacy you want online should be as easy as closing the blinds.
隐私应该是默认的。我们正在建立一个新的在线信任标准，相信在网上获得你想要的隐私应该像关上百叶窗一样容易。

For more privacy advice, follow us on Twitter & get our privacy crash course.
如果您需要更多的隐私建议，请关注我们的推特和获取我们的隐私速成课程。

Dax the duck

We are the Internet privacy company that lets you take control of your information, without any tradeoffs. Welcome to the Duck Side!
我们是互联网隐私公司，让你完全掌控自己的信息。欢迎来到 Duck Side!

Read More 更多
Get a Crash Course on Privacy  获取隐私速成课
Sign up for our newsletter and learn about protecting your online privacy.
注册以获取我们的最新通讯，了解如何保护您的在线隐私。

Your email address will not be shared or associated with anonymous searches. Example Message
您的电子邮件地址将不会与匿名搜索共享或关联。例如：

— DuckDuckGo Blog —
“DuckDuckGo”博客

•DuckDuckGo Testimony Before the United States Senate
  “DuckDuckGo”由美国参议院公证

```

Comment:
This article is not authored by Monero Outreach but is okay. We can leave it but it would need to be changed of _/folder_ (you will have to create the _/archive_ folder).

- Move to folder _/zh-hans/archive_

Reply:
Done.